bin/config-parse.js

```diff
@@ -1,26 +1,74 @@
 #!/usr/bin/env node
 'use strict';
 
+const program = require('commander');
 const winston = require('winston');
-const parser = require('../');
 const fs = require('fs');
 const path = require('path');
 
-const yaml = fs.readFileSync(process.argv[2], 'utf-8');
-const jobName = process.argv[3];
-const destination = process.argv[4];
+const pkg = require('../package');
+const parser = require('../');
 
 winston.remove(winston.transports.Console);
 winston.add(winston.transports.Console, {
     showLevel: false
 });
 
-parser({ yaml, jobName }, (err, data) => {
+let yaml;
+let jobName;
+let permutation;
+
+program
+    .version(pkg.version)
+    .option(
+        '-a, --artifactdir <directory>',
+        'Specify an alternative artifact directory',
+        path.join(process.cwd(), 'artifacts'))
+    .arguments('<screwdriver.yaml> <jobname> <permutation>')
+    .action((yamlFile, job, env) => {
+        jobName = job;
+
+        try {
+            yaml = fs.readFileSync(yamlFile, 'utf-8');
+        } catch (err) {
+            winston.error('Unable to load specified screwdriver.yaml');
+            winston.error(err);
+            process.exit(3);
```

Comment:
We document what each of the different exit codes are. Should we also maintain these magic numbers as constants?